File: 1480. Running Sum of 1d Array/runningSumOf1dArraySolution.cs
/*
Time Complexity - O(n) - Linear Time
Space Complexity - O(1) - Constant Space
*/

public class Solution {
    public int[] RunningSum(int[] nums) {
      
        for(int i = 1; i < nums.Length; i++ )
        {
            // nums[0] is already the correct running sum, so we start loop from i=1
            nums[i] += nums[i-1];
        }
        return nums;
    }
}

File: 1480. Running Sum of 1d Array/runningSumof1dArraySolution.java
class Solution {
    public int[] runningSum(int[] nums) {
        // In Java, array length is accessed via nums.length whereas nums.Length (capital L) in C# 
        for(int i =1; i < nums.length; i++)
        {
            nums[i] += nums[i-1];
        }

        return nums;
    }
}
